### PR TITLE
feat(composer-ui): skeleton of ui

### PR DESCRIPTION
**Pull request by shivenderempire**

### CODE REVIEW

File: apps/composer-ui/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// {
```
Comment (gowravshekar) [inline apps/composer-ui/.eslintrc.js]: Remove the commented block.

File: apps/composer-ui/package-scripts.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  scripts: {
+    default: 'next dev',
```
Comment (gowravshekar) [inline apps/composer-ui/package-scripts.js]: We will sync these similar to the commands of the backend.

File: apps/composer-ui/package.json [hunk 3]
```diff
@@ -3,25 +3,35 @@
...
+    "start": "nps start",
+    "lint": "nps lint",
     "tsc": "tsc",
     "precommit": "lint-staged"
   },
   "dependencies": {
     "@types/node": "20.5.9",
     "@types/react": "18.2.21",
     "@types/react-dom": "18.2.7",
+    "await-to-js": "^3.0.0",
```
Comment (gowravshekar) [inline apps/composer-ui/package.json]: Use this from @app/common.

File: apps/composer-ui/tailwind.config.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: [
+    './app/**/*.{js,ts,jsx,tsx,mdx}',
```
Comment (gowravshekar) [inline apps/composer-ui/tailwind.config.js:4]: Is it possible to use glob patterns to simplify these

**Review by gowravshekar — CHANGES_REQUESTED** [review]

File: apps/composer-ui/tailwind.config.js (hunk 4 above)
Comment (shivenderempire) [reply apps/composer-ui/tailwind.config.js:4]: i haven't done yet, but we can consider later on

File: apps/composer-ui/.eslintrc.js (hunk 1 above)
Comment (shivenderempire) [reply apps/composer-ui/.eslintrc.js]: done

File: apps/composer-ui/package-scripts.js (hunk 2 above)
Comment (shivenderempire) [reply apps/composer-ui/package-scripts.js]: ok

File: apps/composer-ui/package.json (hunk 3 above)
Comment (shivenderempire) [reply apps/composer-ui/package.json]: done

**Review by gowravshekar — APPROVED** [review]

**Review by tahseen0amin — APPROVED** [review]

**Pull request merged**